scripts/stablex/get_auction_elements.js

```diff
@@ -1,14 +1,110 @@
 const BatchExchange = artifacts.require("BatchExchange")
+const BN = require("bn.js")
+const argv = require("yargs")
+  .option("valid", {
+    type: "boolean",
+    describe: "Only show valid and future orders",
+  })
+  .option("covered", {
+    type: "boolean",
+    describe: "Only show orders that have some balance in the sell token",
```

Comment:
For me, covered would mean more than just having some balance, but more like having sufficient balance for the order to be fully filled.

Reply:
You could argue what that means. I was considering do something like "balance at least 1% of the remaining sell amount", but this would fail for our market makers that will have orders with an infinite sell amount.